### PR TITLE
Basic structure

### PR DESCRIPTION
**Pull request by zkr024**
Leaderboard project.
Add a feature branch.
Install Webpack.
Add basic functionality with LocalStorage and modules.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (ShoiraTa) [inline src/index.html:2]: - [ ] You have done a great job setting up webpack, everything works perfectly well 👏🏻 👏🏻 👏🏻 
The only requirement that you still did not meet is that your HTML should look like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). No styling is required on this stage, just make sure to make a similar layout:

Your layout  | Wireframe layout
------------- | -------------
Screenshot 2022-04-12 at 10 32 32 | Screenshot 2022-04-12 at 10 32 18

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @zkr024 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] Please make sure to update the Readme with relevant sections. Currently there is only project title.

- [ ] Check more comments under the review

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (zkr024) [reply src/index.html:2]: @ShoiraTa, Thank you for your feedback, I'll work on it.

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi again @zkr024, 

**Great work on making the changes 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**